app/views/claimToAdjustPoa/AmendablePaymentOnAccount.scala.html

```diff
@@ -87,67 +93,131 @@
+                        )
+                    ),
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(getMessage("table-heading-adjusted-amount.key")),
+                            classes = "govuk-!-width-two-thirds"
+                        ),
+                        value = Value(
+                            classes = "govuk-summary-list__value govuk-table__cell--numeric",
+                            content = HtmlContent(viewModel.paymentOnAccountOne.toCurrencyString)
+                        )
+                    )
+                )
+            } else {
+                Seq(
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(getMessage("table-heading-full-amount.key")),
+                            classes = "govuk-!-width-two-thirds"
+                        ),
+                        value = Value(
+                            classes = "govuk-summary-list__value govuk-table__cell--numeric",
+                            content = HtmlContent(viewModel.paymentOnAccountOne.toCurrencyString)
+                        ),
                     )
                 )
+            },
+            card = Some(Card(
+                title = Some(CardTitle(
+                    content = Text(getMessage("table-heading-1"))
+                )),
+                actions = Some(Actions(
+                    items = Seq(
+                        ActionItem(
+                            href = getChargeSummaryUrl(viewModel.poaOneTransactionId),
+                            content = Text(getMessage("table-heading.link")),
+                            attributes = Map("id" -> "poa1-more-details-date-link")
+                        )
+                    )
+                ))
             ))
-        ))
     ))
 
     @govukSummaryList(SummaryList(
-        rows = Seq(
-            SummaryListRow(
-                key = Key(
-                    content = Text(getMessage("table-heading.key"))
-                ),
-                value = Value(
-                    classes = "govuk-summary-list__value govuk-table__cell--numeric",
-                    content = HtmlContent(paymentOnAccount.paymentOnAccountTwo.toCurrencyString)
-                ),
-            ),
-        ),
-        card = Some(Card(
-            title = Some(CardTitle(
-                content = Text(getMessage("table-heading-2"))
-            )),
-            actions = Some(Actions(
-                items = Seq(
-                    ActionItem(
-                        href = getChargeSummaryUrl(paymentOnAccount.poaTwoTransactionId),
-                        content = Text(getMessage("table-heading.link")),
-                        attributes = Map("id" -> "poa2-more-details-date-link")
+        rows =
+            if(viewModel.poasHaveBeenAdjustedPreviously) {
```

Comment:
not required in scope of this story, but we can refactor this into separate partial functions